auth/credentials/externalaccount/externalaccount.go

```diff
@@ -135,7 +135,7 @@ type ExecutableConfig struct {
 	// This can include arguments. Must be an absolute path for the program. Required.
 	Command string
 	// TimeoutMillis is the timeout duration, in milliseconds. Defaults to 30000 milliseconds when not provided. Optional.
-	TimeoutMillis *int
+	TimeoutMillis int
```

Comment:
Why change TImoutMillis to a pointer?

Reply:
There was no good reason for this type to be a pointer in Go. 0 was always invalid and the value needs to be at least 5 sec or it errors. This will make the type easier to work with. This did not matter before as a pointer since it was an internal detail, but now is being exposed.

Reply:
Sorry, I misread the diff. Good change.